docs/manual/working/scalaGuide/main/sql/slick/PlaySlickAdvancedTopics.md

```diff
@@ -0,0 +1,20 @@
+# Play Slick Advanced Topics
+
+## Connection Pool
+
+With Slick 3.0 release, Slick starts and controls both a connection pool and a thread pool for optimal asynchronous execution of your database actions.
+
+It's important to point out that while Slick allows using a different connection pool than [HikariCP](though, Slick currently only offers built-in support for HikaryCP, and requires you to provide an implementation of [JdbcDataSourceFactory] if you want to use a different connection pool), Play Slick currently doesn't allow it. If you find yourself needing this feature, please drop us a note on the [playframework-dev] mailing list.
+
+> Note: Changing the value of `play.db.pool` won't affect what connection pool Slick is using. In fact, it's worth noting that Play Slick disables both the `play.api.db.DBModule` and `play.api.db.HikariCPModule` Play modules, and it will let Slick be in control of creating and managing the connection pool.
+
+## Thread Pool
+
```

Comment:
Here I should provide  a bit more context on how the thread pool is used by Slick, before discussing tuning.